tests/skeptic.rs

```diff
@@ -6,11 +6,19 @@ use std::process::Command;
 #[test]
 fn readme_test() {
     let rustdoc = Path::new("rustdoc").with_extension(EXE_EXTENSION);
-    let readme = Path::new(file!()).parent().unwrap().parent().unwrap().join("README.md");
+    let readme = Path::new(file!())
```

Comment:
Oh, we should remove this whole file and replace it with the `doc-comment` crate:
https://crates.io/crates/doc-comment (you don't have to do that now, but I just noted this on a PR on another crate of mine so it's in my head).